sdk/core/azure-core/src/main/java/com/azure/core/http/policy/UserAgentPolicy.java

```diff
@@ -62,82 +71,113 @@ public UserAgentPolicy(String userAgent) {
     /**
      * Creates a UserAgentPolicy with the {@code sdkName} and {@code sdkVersion} in the User-Agent header value.
      *
-     * If the passed configuration contains true for AZURE_TELEMETRY_DISABLED the platform information won't be included
-     * in the user agent.
+     * <p>If the passed configuration contains true for AZURE_TELEMETRY_DISABLED the platform information won't be
+     * included in the user agent.</p>
      *
      * @param applicationId User specified application Id.
      * @param sdkName Name of the client library.
      * @param sdkVersion Version of the client library.
-     * @param version {@link ServiceVersion} of the service to be used when making requests.
-     * @param configuration Configuration store that will be checked for the AZURE_TELEMETRY_DISABLED.
-     * @throws NullPointerException if {@code configuration} is {@code null}.
+     * @param configuration Configuration store that will be checked for {@link
+     * Configuration#PROPERTY_AZURE_TELEMETRY_DISABLED}. If {@code null} is passed the {@link
+     * Configuration#getGlobalConfiguration() global configuration} will be checked.
      */
-    public UserAgentPolicy(String applicationId, String sdkName, String sdkVersion, Configuration configuration,
-                           ServiceVersion version) {
-        Objects.requireNonNull(configuration, "'configuration' cannot be null.");
-        boolean telemetryDisabled = configuration.get(Configuration.PROPERTY_AZURE_TELEMETRY_DISABLED, false);
-        if (telemetryDisabled) {
-            this.userAgent = String.format(DISABLED_TELEMETRY_USER_AGENT_FORMAT, sdkName, sdkVersion,
-                version.getVersion());
-        } else {
-            if (applicationId == null) {
-                this.userAgent = String.format(USER_AGENT_FORMAT, sdkName, sdkVersion, getPlatformInfo(),
-                    version.getVersion());
-            } else {
-                this.userAgent = String.format(APPLICATION_ID_USER_AGENT_FORMAT, applicationId, sdkName, sdkVersion,
-                    getPlatformInfo(), version.getVersion());
-            }
-        }
+    public UserAgentPolicy(String applicationId, String sdkName, String sdkVersion, Configuration configuration) {
```

Comment:
So, are we not including the service version in user agent now?

Reply:
This was never being used, while it was used in the String.format calls there weren't enough placeholders for it to be applied.